src/main/java/slimeknights/tconstruct/tools/TinkerModifiers.java

```diff
@@ -142,7 +148,15 @@
   public static final RegistryObject<AutosmeltModifier> autosmelt = MODIFIERS.register("autosmelt", AutosmeltModifier::new);
   public static final RegistryObject<Modifier> expanded = MODIFIERS.register("expanded", () -> new Modifier(0xff9f50));
   public static final RegistryObject<ReachModifier> reach = MODIFIERS.register("reach", ReachModifier::new);
-
+  
+  public static final RegistryObject<BlockTransformModifier> shovelTransform = MODIFIERS.register("shovel_transform", () -> new BlockTransformModifier(0x8a361e, Integer.MIN_VALUE, ToolType.SHOVEL, SoundEvents.ITEM_SHOVEL_FLATTEN, true));
```

Comment:
Minor, but toss in a `// internal abilities` header or something similar here, for consistency. These are "abilities", but not applied via modifiers.

Reply:
They could be, tbh. But sure.